website/docs/components/table/partials/code/how-to-use.md

```diff
@@ -380,6 +380,95 @@ To create a column that has right-aligned content, set `@align` to `right` on bo
 </Hds::Table>
 ```
 
+### Scrollable table
+
+As a general principle, tables should contain as little information as possible. Consuming a large amount of data in a tabular format leads to an intense cognitive load for the user. We recommend using functionalities like [pagination](/components/pagination), [sorting](/components/table?tab=code#sortable-table), and [filtering](/patterns/filter-patterns) to reduce this load.
```

Comment:
> As a general principle, tables should contain as little information as possible.

I think I know what you're trying to say here, but I'm not sure if this lands with me. I read this and think "well, that doesn't make sense, a table can display potentially complex records and data which could mean a ton of information."

Maybe a better way to say this is by leading with the point about usability:

> Consuming a large amount of data in a tabular format can lead to an intense cognitive load for the user. As a general principle, care should be taken to simplify the information within a table as much as possible.

That isn't perfect, but it seems like the guidance should be around simplifying the information and making it human-readible.